nncf/tensorflow/graph/transformations/commands.py

```diff
@@ -21,8 +22,15 @@
 from nncf.common.graph.transformations.commands import TransformationCommand
 from nncf.common.graph.transformations.commands import TransformationPriority
 from nncf.common.graph.transformations.commands import TransformationType
+from nncf.common.stateful_classes_registry import TF_STATEFUL_CLASSES
 
 
+class TFLayerPointStateNames:
+    LAYER_NAME = 'layer_name'
+    TARGET_TYPE = 'target_type'
```

Comment:
Could you please remove this class and use `'layer_name`' and `'target_type'` as keys for consistency with the classes below?

Reply:
the classes below don't use these attributes in `from_state` method.

Reply:
rewrote by using _state_names for CONSISTENCY